### ISSUE TITLE
New name Phone Sync => Phone ?

### ISSUE THREAD

**Issue by nerzhul** [nextcloud/ocsms#153]
Hello,
as you now see app is in nextcloud namespace. I discussed with @BernhardPosselt which prefers renaming app from PhoneSync to Phone because he said "Sync" is implicit.

@stefan-niedermann @enoch85

**Comment by enoch85** [comment]
Cool! 

Yeah, fine by me. :+1:

**Comment by stefan-niedermann** [comment]
Yeap, i'm fine with `Nextcloud Phone`, too :+1:

**Comment by enoch85** [comment]
Just a thought, couldn't this app be confused with the regular Android app if it's called just "Phone"?

**Comment by enoch85** [comment]
cc @AndyScherzinger @tobiasKaminsky

**Comment by AndyScherzinger** [comment]
I am fine with Phone. 

The Android app is called "Nextcloud" as is the iOS app ;)

**Comment by BernhardPosselt** [comment]
> Just a thought, couldn't this app be confused with the regular Android app if it's called just "Phone"?

You won't call the app Phone on Android. You will call it something like Nextcloud/ownCloud Phone Connector or something similar. This name will only be used for the Nextcoud app in PHP since it fits the general convention pretty well: Notes, News, Calendar, Contacts, Files, Gallery, ... all one word that describes what they are used for ;)

**Comment by nerzhul** [comment]
Agree with Bernhard

**Comment by enoch85** [comment]
I rest my case.

**Comment by JSoko** [comment]
Hello,

I think anythink with phone isn't a good name as long as the app "just" handles sms.

Why not just name it "Nextcloud SMS"?

**Comment by BernhardPosselt** [comment]
@JSoko because the plan is to add more features than SMS which interface with your Phone

**Comment by enoch85** [comment]
What he said ---^

**Comment by AndyScherzinger** [comment]
Well, it still needs to be shortened and the App shouldn't have Nextcloud in the same, simply since you can't read it.. Hello Phone screen ;)
![device-2017-01-30-192816](https://cloud.githubusercontent.com/assets/1315170/22435983/7174a9d8-e722-11e6-83c9-a5587a0d34e8.png)
Now it looks like Nextcloud Messenger :P

**Comment by stefan-niedermann** [comment]
Yeah, but you have to differ between the **project name** and the **launcher name** - the notes app is called "Nextcloud Notes" while the launcher name shows only "Notes".
So "Nextcloud Phone Sync" would be good and the launcher name could only be "Phone Sync" - in combination with a blue/white icon everyone who installed it will know it belongs to Nextcloud ;)

**Comment by BernhardPosselt** [comment]
@stefan-niedermann Android apps are a different topic, there can be more than one app that interfaces with the Nextcloud app with different names etc.

**Comment by awikatchikaen** [comment]
Caution with the name.
My father has an account on my Nextcloud instance and had a contact's sync problem and was looking around the app "phone sync" on my instance. 

"Phone" could be confusing too

**Comment by enoch85** [comment]
Hey why not "iPhone"? :D

**Comment by peturbg** [comment]
calm down. dont you see you are attacking yourself for something stupid and on this planet earth peoples dont have what to eat and die. nextcloud sync sms and if it need to be short nextcloud sms. nextcloud have dissadvantatge, when there is new version of program there is no changelog. in nextcloud apps there is no changelog

**Comment by BernhardPosselt** [comment]
Change log support it is built in but apps need to provide one. Also please stay on topic and don't hijack issues like you did in passman

**Comment by dienteperro** [comment]
I agree with @BernhardPosselt Phone is fine.

**Comment by azmeuk** [comment]
Yep. Phone is great!

**Comment by UshurX** [comment]
That's cool. So I can use Phone for making calls? That's great. So NextCloud get's a bit more like the MS Cloud with Teams. Will be SMS possible, too?!
